lib/ndr_import/xml/table.rb

```diff
@@ -179,6 +181,12 @@ def mappable_xpaths_from(line)
         xpaths
       end
 
+      def populated_leaf?(node)
+        node.element_children.empty? &&
```

Comment:
is this useable? https://www.rubydoc.info/github/sparklemotion/nokogiri/Nokogiri%2FXML%2FNode:blank%3F

Reply:
I don't _think_ so

Given this xml, we'd expect `pathology` to be blank:

```
<root>
  <record>
    <no_relative_path value="A value"/>
    <no_path_or_att>Another value</no_path_or_att>
    <demographics>
      <demographics_1>AAA</demographics_1>
      <demographics_2 code="03">Inner text</demographics_2>
      <address></address>
    </demographics>
    <pathology></pathology>
  </record>
</root>
```

But `blank?` returns false.
```
[1] pry(#<NdrImport::Xml::Table>)> node
=> #(Element:0x1360 { name = "pathology" })
[2] pry(#<NdrImport::Xml::Table>)> node.blank?
=> false
```